### PR TITLE
[Feed Storage]setting up post_wilting_moisture_percentage in baleage is an user input.

### PR DESCRIPTION
**Pull request by matthew7838**
Adds the ability for the user to input for post_wilting_moisture_percentage.

### Context

Issue(s) closed by this pull request: closes #

### What

- Changed the naming of final_moisture_percentage to post_wilting_moisture_percentage as Kevin fins the naming to be confusing.
- Added the current value in the default feed management input file.
- Set up default and min-max advised by Kevin for this variable.

### Why

Allow user to set post_wilting_moisture_percentage.

### How

Users will now be able to have their own desirable post_wilting_moisture_percentage as inputs.

### Test plan

Run e2e testing with the print statement and make sure that the values are read correctly from the input file.

### CODE REVIEW

File: RUFAS/routines/feed_storage/baleage.py [hunk 1]
```diff
@@ -30,6 +26,9 @@ class Baleage(Storage):
 
     def __init__(self, capacity: float = float("inf")):
         super().__init__(capacity)
+        im = InputManager()
+        self.post_wilting_moisture_percentage: float = im.get_data("feed_management.post_wilting_moisture_percentage")
+        print(self.post_wilting_moisture_percentage)
```
Comment (Isgotkowitz) [inline RUFAS/routines/feed_storage/baleage.py]:
```suggestion
```
Don't forget to remove this print statement

File: RUFAS/routines/feed_storage/baleage.py [hunk 2]
```diff
@@ -3,16 +3,12 @@
...
 INITIAL_LOSS_PERIOD = 30
 
 
-"""Defines the final moisture level that baleage will dry down to. TODO: make this value a user input - #2038"""
-DEFAULT_FINAL_MOISTURE_PERCENTAGE = 50.0
-
-
 class Baleage(Storage):
     """
     Class representing Baleage storage, a subclass of Storage.
```
Comment (Isgotkowitz) [inline RUFAS/routines/feed_storage/baleage.py:14]: Could you please add `post_wilting_moisture_percentage` as one of the attributes of baleage?

**Review by Isgotkowitz — APPROVED** [review]
LGTM, left comments on some small things that could be improved.

Also, please don't forget to link and close the issue that this PR addresses

File: input/metadata/end_to_end_testing_metadata.json [hunk 3]
```diff
@@ -188,7 +188,14 @@
...
+    },
+    "feed_management": {
+      "title": "Feed Management Dataset",
+      "description": "Feed management data.",
+      "path": "input/data/feed_management/default_feed_management.json",
+      "type": "json",
+      "properties": "feed_storage_input_properties"
+    },
+    "cross-validation": {}
+  }
```
Comment (ew3361zh) [inline input/metadata/end_to_end_testing_metadata.json]: I think `cross-validation` needs to be outside the close-parenthesis directly below it

**Review by ew3361zh — APPROVED** [review]
Looks good. e2e testing was failing for me as this PR stands right now but I moved the `cross-validation` section of the "end_to_end_testing_metadata.json" file outside the `files` section and it worked fine.
Please fix that but otherwise looks great!

**Pull request merged**